source/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,114 @@
+//
+// Footer Component
+//
+// Based on the existing GOV.UK footer in GOV.UK Template
+
+@include exports("footer") {
+
+  .app-c-footer {
+    display: block;
+    margin-top: $govuk-spacing-scale-1;
+    padding: $govuk-gutter $govuk-gutter-half ($govuk-gutter * 2);
+    border-top: 1px solid $govuk-footer-border-top;
+    color: $govuk-footer-text;
+    background: $govuk-footer-background;
+    @include mq($from: desktop) {
+      padding: $govuk-gutter;
+    }
+
+    a:link,
+    a:visited {
+      color: $govuk-footer-link;
+    }
+
+    a:hover,
+    a:active {
+      color: $govuk-footer-link-hover;
+    }
+  }
+
+  .app-c-footer__heading {
+    margin-bottom: 0;
+    color: $govuk-text-colour;
+  }
+
+  .app-c-footer__heading:nth-of-type(1) {
+    margin-top: 0;
+  }
+
+  .app-c-footer__list {
+    margin: 0;
+
+    li {
+      margin-bottom: 0;
+    }
+
+    a {
+      display: block;
+      padding: $govuk-spacing-scale-2 0 0;
+      @include govuk-core-16;
+    }
+  }
+
+  .app-c-footer__licence {
+    line-height: 1;
+    @include mq ($from: desktop) {
+      display: inline-block;
+      width: 75%;
+      vertical-align: bottom;
+    }
+  }
+
+  .app-c-footer__licence-logo {
+    display: block;
+    width: 41px; // Based on the open-government-licence_2x.png image width: 81px/2
+    height: 17px; // Based on the open-government-licence_2x.png image height: 33px/2
+    margin-top: -2px; // Align baseline with paragraph
```

Comment:
Is this correction for the font? Can we group these at the bottom of the file and clearly signpost them [like this](https://github.com/alphagov/govuk-design-system/blob/master/source/stylesheets/components/_header.scss#L100-L113)?

Reply:
Missed that one. No, it was a way to align the license text on the left with the copyright text on the right, but there's no need to keep the `margin-top` in the new footer layout so I just removed the line (also tested on all browsers to make sure doesn't affect anything).